sdk/bootstrap/main.go

```diff
@@ -330,48 +331,43 @@ func isFile(path string) (bool, error) {
 }
 
 func GetLocalIP() string {
-    addrs, err := net.InterfaceAddrs()
-    if err != nil {
-        return ""
-    }
-    for _, address := range addrs {
-        // check the address type and if it is not a loopback the display it
-        if ipnet, ok := address.(*net.IPNet); ok && !ipnet.IP.IsLoopback() {
-            if ipnet.IP.To4() != nil {
-                return ipnet.IP.String()
-            }
-        }
-    }
-    return ""
+	ip, err := mesos.ContainerIP()
+
+	if err != nil {
+		return ""
+	}
+
+	return ip.String()
 }
+
 // main
 
 func main() {
 	args := parseArgs()
 
-    libprocess_ip, found := os.LookupEnv("LIBPROCESS_IP")
+	libprocess_ip, found := os.LookupEnv("LIBPROCESS_IP")
```

Comment:
there are now two fetches of LIBPROCESS_IP, one here and one in `ip.go`. should this one go away?